app/services/riiif/imagemagick_command_factory.rb

```diff
@@ -39,7 +43,7 @@ def build
     private
 
       def command
```

Comment:
Makes sense. As a public API, do you prefer 'command' or 'external_command'? We can avoid the redirection either way, since as you helpfully pointed out it was just a private method anyway.